lightspeed/modules/proc_remove-seat-licenses.adoc

```diff
@@ -5,10 +5,12 @@
 = Removing seat licenses
 
 [role="_abstract"]
-You can remove users who no longer need to access {LightspeedShortName}, which makes licenses available for others to use.
-[NOTE]
+Remove users who no longer need to access {LightspeedShortName}, which makes licenses available for others to use. When you remove a user, the Ansible user's {LightspeedShortName} seat license only is removed; it does not remove the user from your Red Hat Customer Portal organization. 
```

Comment:
With this "You can" gone, the sentence doesn't really read right. How about: "You can remove seat assignments from users who no longer need to access {LightspeedShortName}, making those seats available for others to use. When you remove a user from the Seat Management tool, the Ansible user's {LightspeedShortName} seat license only is removed; it does not remove the user from your Red Hat Customer Portal organization."

Reply:
Readded 'you can', rest text is the same. TY.